user_api_app/permissions.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework import permissions
+
+
+class UpdateOwnProfile(permissions.BasePermission):
```

Comment:
We have not used this anywhere we can use this in `CustomUserList `

see `permission_classes`